amethyst_core/src/transform/components/transform.rs

```diff
@@ -560,6 +561,31 @@ impl Transform {
             .to_homogeneous()
             .append_nonuniform_scaling(&inv_scale)
     }
+
+    /// Calculates the inverse of this transform, which is in effect the 'view matrix' as
+    /// commonly seen in computer graphics. This function computes the view matrix for the
+    /// global transformation of the entity, and so takes into account Parents.
+    ///
+    /// We can exploit the extra information we have to perform this inverse faster than `O(n^3)`.
+    pub fn global_view_matrix(&self) -> Matrix4<Float> {
```

Comment:
Can you please add a few tests for this method?

Reply:
Done. I added another test which matches the view_matrix test but performs it on the global view matrix.